android/app/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,7 @@
         android:exported="true"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:launchMode="singleTask"
-        android:windowSoftInputMode="adjustResize">
+        android:windowSoftInputMode="adjustPan">
```

Comment:
This works, though it doesn't seem to be recommended on Android according to https://developer.android.com/guide/topics/manifest/activity-element. I don't think that needs to block merge, though. Why did you use this approach as opposed to, say, [`KeyboardAvoidingView`](https://reactnative.dev/docs/keyboardavoidingview)?

Reply:
I tried using `KeyboardAvodingView `without changing any value in the `Android manifest `, but that didn't make the keyboard stay and it added some default padding to it that made the TextInput appear on the top. So in order to make  the keyboard pop open once the user clicks on the TextInput I had to change the value to `adjustPan`.

Reply:
Fair, I'm just worried about other consequences of this change. I'm not seeing any right now, just wanted to hear that you explored what seems like the more conventional approach.